include/sundials/sundials_adaptcontroller.h

```diff
@@ -31,16 +31,17 @@ extern "C" {
 #endif
 
 /* -----------------------------------------------------------------
- * SUNAdaptController types (currently, only "H" is implemented;
- * others are planned):
+ * SUNAdaptController types:
  *    NONE - empty controller (does nothing)
  *    H    - controls a single-rate step size
+ *    MRI_TOL - controls slow and fast relative tolerances
  * ----------------------------------------------------------------- */
 
 typedef enum
 {
   SUN_ADAPTCONTROLLER_NONE,
-  SUN_ADAPTCONTROLLER_H
+  SUN_ADAPTCONTROLLER_H,
+  SUN_ADAPTCONTROLLER_MRI_TOL
```

Comment:
I don't see any fundamental flaw with the current name, or need to change it to one of the suggestions.  If others feel strongly then I'll reconsider.

Reply:
I suggested `MRI_H_TOL` since that's what it is referred to in the docs. However, if we foresee using this with other, non-MRI methods (e.g., splitting methods) then maybe we want to drop or change `MRI` since it wouldn't be specific to MRI methods.